resources/META-INF/plugin_template.xml

```diff
@@ -75,6 +75,10 @@
               text="Flutter Packages Upgrade"
               description="Run 'flutter packages upgrade'"/>
       <separator/>
+      <action id="flutter.clean" class="io.flutter.actions.FlutterCleanAction"
+              text="Flutter Clean"
+              description="Run 'flutter clean'"/>
```

Comment:
I'm not sure if the description shows up anywhere but if it does then something more descriptive would be nice.
"Remove build artifacts" perhaps.

Reply:
It doesn't show anywhere as far as I can see but you make a good point.  All of the action descriptions should be more descriptive.  I've made a note to follow up on that.